src/projections.F90

```diff
@@ -56,7 +56,7 @@ subroutine point_curve(x0, t, k, coef, nctl, ndim, Niter, eps, s, Diff)
 
   ! Working
   real(kind=realType)   :: val(ndim), deriv(ndim), deriv2(ndim), step, c, dist, p_diff
-  real(kind=realType)   :: grad, hessian,  update, R(ndim), nDist, fval, nfval, pgrad, newPt
+  real(kind=realType)   :: grad, temp(1), hessian,  update, R(ndim), nDist, fval, nfval, pgrad, newPt
```

Comment:
Is this an okay solution? I create a temporary singleton array and use this to pass value to the subroutine, as recommended [here](https://gcc.gnu.org/gcc-10/porting_to.html).

Reply:
This seems okay to me.

Reply:
In general I am not a fan, but this is the simplest. Would need to think for a better solution.

Reply:
I removed this and replaced by the `norm` invocation with `ABS` which is equivalent for scalars.